compliance/azure/azure_low_cpu_usage/README.md

```diff
@@ -0,0 +1,27 @@
+## Azure Below CPU Utilization
+ 
+### What it does
+This policy checks all the instances in the Azure Subscription for the average CPU usage over the the last hour.  If the usage is less than the user provided CPU percentage threshold then the virtual machines are recomended for removal, and the user is emailed.
+
+### Functional Details
+ 
+The policy leverages the Azure API to check all instances and then grab the utilization over the past hour, finally recomendending the low ones for decomissioning
+ 
+#### Input Parameters
+ 
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+- *CPU Threshold* - Percentage of CPU utilization
+- *Azure Subscription ID* - Your Azure Subscription ID.  You can find it by following this guide: https://docs.bitnami.com/azure/faq/administration/find-subscription-id/
```

Comment:
Check other azure policies for this parameter and match them.

Reply:
It seems to match